### ISSUE TITLE
Avancé : parler des registres privés

### ISSUE THREAD

**Issue by oncletom** [oncletom/nodebook#405]
Et qu'on peut utiliser Gitlab, GitHub, Frogtruc, private npm ou même hoster le sien avec `verdaccio` (successeur de sinopia si j'ai bien compris).

**Comment by juanpicado** [comment]
Yes, that's correct, you can read the complete story here https://medium.com/verdaccio/the-crazy-story-of-verdaccio-d35d924728bc
